src/components/pager/__internal__/pager-navigation-link.component.tsx

```diff
@@ -61,30 +88,22 @@ const PagerNavigationLink = ({
     <StyledPagerLinkStyles
```

Comment:
Not essential that you change it as it's not a change you've made, but the name of this styled component is a little strange I think it would be fine to just call it `StyledPagerLink`